src/components/organisms/DropdownContent/DropdownContent.tsx

```diff
@@ -0,0 +1,151 @@
+import { Dispatch, FC, SetStateAction, useState } from 'react'
+import classNames from 'classnames'
+import { includes, map } from 'lodash'
+import CheckBoxInput from 'components/molecules/CheckBoxInput/CheckBoxInput'
+import Button, {
+  AppearanceTypes,
+  SizeTypes,
+} from 'components/molecules/Button/Button'
+import { SelectionControlsInputProps } from 'components/organisms/SelectionControlsInput/SelectionControlsInput'
+
+import classes from './styles.module.scss'
+
+type DropdownContentProps = SelectionControlsInputProps & {
+  isOpen?: boolean
+  selectedOptionObjects?: { label: string; value: string }[]
+  setIsOpen?: Dispatch<SetStateAction<boolean>>
+}
+
+const DropdownContent: FC<DropdownContentProps> = ({
+  dropdownSize = 'l',
+  disabled,
+  isOpen,
+  searchInput,
+  options,
+  multiple = false,
+  value,
+  name,
+  ariaLabel,
+  buttons = false,
+  cancelButtonLabel,
+  proceedButtonLabel,
+  onChange,
+  setIsOpen,
+  helperText,
+  selectedOptionObjects,
+  tags,
+}) => {
+  const [selectedValue, setSelectedValue] = useState<string[]>(
+    value ? (Array.isArray(value) ? value : [value]) : []
+  )
+
+  const handleSingleSelect = (selectedOption: string) => {
+    onChange(selectedOption ? selectedOption : '')
+  }
+
+  const handleMultipleSelect = (selectedOption: string) => {
+    const selectedValues = Array.isArray(selectedValue)
+      ? [...selectedValue]
+      : []
+    const optionIndex = selectedValues.indexOf(selectedOption)
+
+    const newSelectedValues =
+      optionIndex === -1
+        ? [...selectedValues, selectedOption]
+        : selectedValues.filter((value) => value !== selectedOption)
+
+    setSelectedValue(newSelectedValues)
+
+    if (setIsOpen) {
+      setIsOpen(true)
+    }
+  }
+
+  const handleOnSave = () => {
+    onChange(selectedValue)
+  }
+
+  const handleCancel = () => {
+    setSelectedValue([])
+  }
+
+  return (
+    <>
+      <div
+        className={classNames(classes.dropdownMenu, classes[dropdownSize])}
+        hidden={disabled || !isOpen}
+      >
+        <div hidden={!searchInput}>{searchInput}</div>
+
+        <ul>
+          {map(options, (option, index) => {
+            const isMultiSelected =
+              selectedValue && includes(selectedValue, option?.value)
+            const isSingleSelected = value && includes(value, option?.value)
+
+            return (
+              <li key={index} className={classes.dropdownMenuItem}>
+                {multiple && (
+                  <CheckBoxInput
+                    name={name}
+                    ariaLabel={ariaLabel}
```

Comment:
**[blocker]**

Not ideal, this would mean that every option has the same ariaLabel, which can confuse users who rely on this a lot. Essentially for them every option would be the same. Rather use the option.label value for this